db/models/introspect.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  var Introspect = sequelize.define('Introspect', {
+    name: DataTypes.STRING,
```

Comment:
Do we need id in the introspect model?

Reply:
Sequelize will add this automatically when sync'ing the table (along with created/updated fields). We can be more explicit about this.